src/CMakeLists.txt

```diff
@@ -58,12 +58,17 @@ endif(WIN32)
 #######################################################################################################################
 # Build the program
 add_executable(${PROJECT_NAME} ${${PROJECT_NAME}_SOURCE} ${UIS_HDRS} ${RESOURCES_LIST})
-qt5_use_modules(${PROJECT_NAME} Core Network Widgets Svg)
```

Comment:
Why change this? Even is the other way is preferred, is the change related to the porting?